### ISSUE TITLE
Bioconductor

### ISSUE THREAD

**Issue by zz327** [greenelab/computational-reagents#13]
Bioconductor provides tools for the analysis and comprehension of high-throughput genomic data.
It helps - Everyone could download and re-run the program in R language;
             - It usually comes with test example and Reference Manual
             - Anybody who had problem with those package can submit a report and Bioconductor would directly contact the author of the package
However, not all of the code are open source, which is quite disappointing.

**Comment by linzho** [comment]
I agree that Bioconductor is really useful - if paper writers would cite exactly what version of the package they are using and in what R version, and Bioconductor had one central place to locate all such versions, it would really help reproducibility

**Comment by sklasfeld** [comment]
I like the vignettes in for Bioconductor packages. Similar to Linda, I have found that many papers will just say we used "Diffbind" (a bioconductor package) and not explain what they did with the package, which version they used etc. Therefore, though bioconductor is nice for providing reproducible tools, it seems to not promote reproducibility.
